### PR TITLE
New models for softmax

### PR DESCRIPTION
**Pull request by pobonomo**
Do new models for softmax

### CODE REVIEW

**Comment by mmaaz-git** [comment]
Hi Pierre: just wanted to check if you have an estimate for when you think this will be merged. I'd love to use it! No pressure at all. Or as you indicate it's safe to test, so I could locally clone this PR and play around with it? Thanks.

**Comment by pobonomo** [comment]
> Hi Pierre: just wanted to check if you have an estimate for when you think this will be merged. I'd love to use it! No pressure at all. Or as you indicate it's safe to test, so I could locally clone this PR and play around with it? Thanks.

Hi,

I have no estimate currently. The models are working I think but it's a big change and there is a lot of cleanup.

Not that what is implemented so far is using softmax for scikit-learn only: multinomial logistic regression and neural networks. I didn't implement counter-parts for Keras and Pytorch.

I think that it is usable though so if you want to play with it you could try (and more testing cannot hurt).

Best,
Pierre

**Comment by mmaaz-git** [comment]
Thanks, will do!
